libr/debug/p/debug_native.c

```diff
@@ -338,75 +337,14 @@ static RList *r_debug_native_pids (int pid) {
 			if (p) r_list_append (list, p);
 		}
 	}
+#elif __linux__
+	return linux_pid_list (pid, list);
 #else
-	int i, fd;
```

Comment:
linux_procfs is necessary for running linux bins on BSD systems, and it should be compatible with this parser